### PR TITLE
[SPARK-28187][build] Add support for hadoop-cloud to the PR builder.

### PR DESCRIPTION
**Pull request by vanzin**

### CODE REVIEW

File: hadoop-cloud/pom.xml [hunk 1]
```diff
@@ -28,11 +28,9 @@
 
   <artifactId>spark-hadoop-cloud_2.12</artifactId>
   <packaging>jar</packaging>
-  <name>Spark Project Cloud Integration through Hadoop Libraries</name>
+  <name>Spark Project Hadoop Cloud Integration</name>
```
Comment (vanzin) [inline hadoop-cloud/pom.xml]: I'm changing this file mostly to test the PR builder. But also to remove some verbosity...

**Comment by vanzin** [comment]
retest this please

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.
Both Sbt/Maven builds are triggered with `hadoop-cloud` module as expected.

**Comment by vanzin** [comment]
Merging to master.

**Pull request closed without merging**